Source/Core/Core/Boot/Boot.cpp

```diff
@@ -475,5 +475,19 @@ bool CBoot::BootUp()
 	// Not part of the binary itself, but either we or Gecko OS might insert
 	// this, and it doesn't clear the icache properly.
 	HLE::Patch(0x800018a8, "GeckoCodehandler");
+
+	u32 argvMagic = Memory::Read_U32(PC + 4);
```

Comment:
This should likely also be moved to its own function, so it has a self-documenting name.